### PR TITLE
fix: include native-image files correctly

### PR DESCRIPTION
**Pull request by klopfdreh**
Fixes: https://github.com/JodaOrg/joda-time/issues/783

Hint: I added a new resource block with **/*.json as there may also be a reflect-config.json in future which might also be included.

Test result:
Bildschirmfoto 2024-09-04 um 06 04 35

## Summary by CodeRabbit

- **New Features**
	- Added a new contributor, Tobias Soloschenko, to acknowledge their contributions.
	- Expanded resource management by including specific JSON files from a new directory in the build process.

These changes enhance project documentation and improve resource handling without affecting existing functionality.

### CODE REVIEW

**Comment by jodastephen** [comment]
@coderabbitai Why do you say that "The file reflect-config.json is not correctly copied from META-INF/native-image to target"?

File: pom.xml [hunk 1]
```diff
@@ -369,6 +369,10 @@
       <name>Alex</name>
       <url>https://github.com/nyrk</url>
     </contributor>
+    <contributor>
+      <name>Tobias Soloschenko</name>
```
Comment (jodastephen) [inline pom.xml:373]: Can you add this alphabetcally by surname please.

File: pom.xml [hunk 2]
```diff
@@ -393,6 +397,13 @@
   <!-- ==================================================================== -->
   <build>
     <resources>
+      <resource>
+        <directory>${project.basedir}/src/main/resources/META-INF</directory>
+        <targetPath>META-INF</targetPath>
+        <includes>
+          <include>**/*.json</include>
```
Comment (jodastephen) [inline pom.xml:404]: I would prefer a more precise definition here - I'm guessing that would be `native-image/joda-time/joda-time/resource-config.json`

**Review by jodastephen — CHANGES_REQUESTED** [review]

**Comment by klopfdreh** [comment]
@jodastephen changes are made and force pushed as a single commit to merge directly. 👍

**Review by jodastephen — APPROVED** [review]

**Pull request merged**